src/CSS/Home.css

```diff
@@ -1,17 +1,16 @@
 .clubTitleSection {
   position: relative;
   width: 100%;
-  height: 23vw;
+  height: 26vw;
```

Comment:
since some of the photos are being cut off, can you make this height value slightly bigger